examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -80,14 +80,14 @@
   {
```

Comment:
<p>We should store this to disk with .to_parquet.</p><p> </p><p>The NVTabular runtime would be to measure this cell including storing it to disk</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>